projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -140,6 +134,13 @@ public static function load_wpcom_user_features() {
 		require_once __DIR__ . '/features/wpcom-sidebar-notice/wpcom-sidebar-notice.php';
 		require_once __DIR__ . '/features/wpcom-site-management-widget/class-wpcom-site-management-widget.php';
 		require_once __DIR__ . '/features/wpcom-themes/wpcom-themes.php';
+
+		// Only load the Calypsoify and Masterbar features on WoA sites.
+		if ( class_exists( '\Automattic\Jetpack\Status\Host' ) && ( new \Automattic\Jetpack\Status\Host() )->is_woa_site() ) {
+			\Automattic\Jetpack\Calypsoify\Jetpack_Calypsoify::get_instance();
+			// This is temporary. After we cleanup Masterbar on WPCOM we should load Masterbar for Simple sites too.
+			\Automattic\Jetpack\Masterbar\Main::init();
```

Comment:
I assume these 2 are wpcom user features, right 🤔

Reply:
Yes, but I didn't want to make too many changes in the last PR (https://github.com/Automattic/jetpack/pull/38364)

Are this changes really necessary here?

Reply:
At least it should be called after the flag defined.

Reply:
Ohhhhhh.... I seee....